pkg/client/cli/cmd_list.go

```diff
@@ -33,6 +35,7 @@ func listCommand() *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.BoolVarP(&s.onlyIntercepts, "intercepts", "i", false, "intercepts only")
+	flags.BoolVarP(&s.allIntercepts, "all-intercepts", "A", false, "gets all live intercepts, overrides all other flags")
```

Comment:
We never want to search through all workloads in all namespaces. I could make do the above and also change the implementation so that if `-A` is set, `-i` or `-a` must also be set, or throw an error.